### PR TITLE
pd: update placement rules

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What is changed, added or deleted? (Required)

- add the usage of the new add bundle API
- remove the old load/save method

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Other reference link(s): https://github.com/tikv/pd/pull/2927 https://github.com/tikv/pd/issues/2865

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: configure-placement-rules.md [hunk 1]
```diff
@@ -263,6 +245,74 @@ pd-ctl config placement-rules load --group=pd --out=rule.txt
     pd-ctl config placement-rules rule-group delete pd
     ```
 
+### 使用 pd-ctl 批量更新分组及组内规则
+
+使用 `rule-bundle` 子命令，可以比较方便地同时查看修改规则分组及组内的所有规则。
```
Comment (TomShawn) [inline configure-placement-rules.md:250]:
```suggestion
使用 `rule-bundle` 子命令，可以方便地同时查看和修改规则分组及组内的所有规则。
```

File: configure-placement-rules.md [hunk 2]
```diff
@@ -263,6 +245,74 @@ pd-ctl config placement-rules load --group=pd --out=rule.txt
...
+      "start_key": "",
+      "end_key": "",
+      "role": "voter",
+      "count": 3
+    }
+  ]
+}
+```
+
+`rule-bundle get` 命令可以使用 `-out` 参数来将输出写入文件，方便后续修改保存。
```
Comment (TomShawn) [inline configure-placement-rules.md:280]:
```suggestion
`rule-bundle get` 子命令中可以添加 `-out` 参数来将输出写入文件，方便后续修改保存。
```

File: configure-placement-rules.md [hunk 3]
```diff
@@ -263,6 +245,74 @@ pd-ctl config placement-rules load --group=pd --out=rule.txt
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+pd-ctl config placement-rules rule-bundle get pd -out="group.json"
+```
+
+修改完成后，使用 `rule-bundle set` 命令将文件中的配置保存至 PD 服务器。与前面介绍的 `save` 不同，此命令会替换服务器端该分组内的所有规则。
```
Comment (TomShawn) [inline configure-placement-rules.md:288]:
```suggestion
修改完成后，使用 `rule-bundle set` 子命令将文件中的配置保存至 PD 服务器。与前面介绍的 `save` 不同，此命令会替换服务器端该分组内的所有规则。
```

File: configure-placement-rules.md [hunk 4]
```diff
@@ -263,6 +245,74 @@ pd-ctl config placement-rules load --group=pd --out=rule.txt
     pd-ctl config placement-rules rule-group delete pd
     ```
 
+### 使用 pd-ctl 批量更新分组及组内规则
+
+使用 `rule-bundle` 子命令，可以比较方便地同时查看修改规则分组及组内的所有规则。
+
+其中 `get {group_id}` 用来查询一个分组，输出为嵌套形式的规则分组和组内规则：
```
Comment (TomShawn) [inline configure-placement-rules.md:252]:
```suggestion
该子命令中 `get {group_id}` 用来查询一个分组，输出结果为嵌套形式的规则分组和组内规则：
```

File: configure-placement-rules.md [hunk 5]
```diff
@@ -263,6 +245,74 @@ pd-ctl config placement-rules load --group=pd --out=rule.txt
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+pd-ctl config placement-rules rule-bundle set pd -in="group.json"
+```
+
+### 使用 pd-ctl 查看和修改所有配置
+
+最后一种直观的修改配置的方式是把全部配置保存至文件，修改后再覆盖保存。我们同样使用 `rule-bundle` 子命令。
```
Comment (TomShawn) [inline configure-placement-rules.md:298]:
```suggestion
用户还可以使用 pd-ctl 查看和修改所有配置，即把全部配置保存至文件，修改后再覆盖保存。该操作同样使用 `rule-bundle` 子命令。
```

**Comment by disksing** [comment]
Please help take a look @Yisaer

**Review by Yisaer — APPROVED** [review]
LGTM

**Pull request merged**